lib/Auth/OAuth.php

```diff
@@ -363,7 +363,7 @@ public function isAuthorized()
         }
 
         //Check to see if token in session has expired
-        if (!empty($this->_expires) && $this->_expires < time()) {
+        if (true || !empty($this->_expires) && $this->_expires < time()) {
```

Comment:
Isn't the `(true || ` part a remnant from testing? If not, let's remove the if since it will be true every time.